examples/generic_connector_configs/dropbox_secretless.yml

```diff
@@ -0,0 +1,48 @@
+version: 2
```

Comment:
This is probably outside of the scope of this commit, but could we consider adding unit tests for this config using a DropBox server container and a DropBox client container? I don't know if the available DropBox server containers support all of the authentication modes that you've added here.